libs/runtime/ebpf_ring_buffer.h

```diff
@@ -8,7 +8,17 @@
 
 CXPLAT_EXTERN_C_BEGIN
 
-typedef struct _ebpf_ring_buffer ebpf_ring_buffer_t;
+typedef struct _ebpf_ring_buffer
```

Comment:
non blocking: consider creating a issue to make this struct opaque with "getter" functions. You can resolve this comment.